core/domain/learner_progress_services.py

```diff
@@ -616,3 +648,118 @@ def get_collection_summary_dicts(collection_summaries):
         })
 
     return summary_dicts
+
+
+def get_activity_progress(user_id):
+    """Returns the progress of the learners - the explorations and collections
+    completed by the user and those in progress.
+
+    Args:
+        user_id: str. The id of the learner.
+
+    Returns:
```

Comment:
Should I just keep the return type as a tuple? We are just returning a set of summaries (and a few numbers).

What do you think?